src/components/dashboard/sidebar/utils.test.js

```diff
@@ -30,28 +31,48 @@ describe('getOfferExpiringFirst', () => {
 
 describe('getPolicyExpiringFirst', () => {
   it.each([
-    { policies: undefined },
-    { policies: [] },
+    {
+      policies: {
+        expiredPolicies: [],
+        unexpiredPolicies: [],
+      },
+    },
+    {
+      policies: {
+        expiredPolicies: undefined,
+        unexpiredPolicies: undefined,
+      },
+    },
   ])('returns undefined if no policies', ({ policies }) => {
     expect(getPolicyExpiringFirst(policies)).toBeUndefined();
   });
-
-  it('returns undefined if no current policies', () => {
-    const policies = [{ active: false }];
-    expect(getPolicyExpiringFirst(policies)).toBeUndefined();
-  });
-
   it('returns policy expiring first', () => {
-    const policies = [{
-      subsidyExpirationDate: '2022-01-01T00:00:00Z',
+    const policies = {
+      expiredPolicies: [{
+        subsidyExpirationDate: dayjs().subtract(1, 'days').toISOString(),
+        active: false,
```

Comment:
[inform] in theory, `credits_available` should never return a policy with `active: false`, but can't hurt to handle handle it explicitly either way I guess.